lib/linguist/heuristics.yml

```diff
@@ -113,6 +113,12 @@ disambiguations:
     #  : dependency
     # path/file.ext1 : some/path/../file.ext2
     pattern: '([\/\\].*:\s+.*\s\\$|: \\$|^[ %]:|^[\w\s\/\\.]+\w+\.\w+\s*:\s+[\w\s\/\\.]+\w+\.\w+)'
+- extensions: ['.dsp']
```

Comment:
Could you also add a test in [`test/test_heuristics.rb`](https://github.com/github/linguist/blob/master/test/test_heuristics.rb) for these heuristic rules?

Reply:
This is done, and tried on my side, looking fine.